src/frontend/context/ManuellJournalføringContext.tsx

```diff
@@ -202,6 +203,44 @@ const [ManuellJournalføringProvider, useManuellJournalføring] = createUseConte
         }
     };
 
+    /*
+    *****************************************************************************
+    Dokument Visning - temporary feature
+    */
+    const [valgtDokumentId, settValgtDokumentId] = React.useState<string | undefined>(undefined);
+    const [dokumentData, settDokumentData] = React.useState(byggTomRessurs<string>());
+    const [visDokument, settVisDokument] = React.useState(false);
+
+    const hentDokumentData = (journalPostId: string, dokumentInfoId: string) => {
+        settDokumentData(byggHenterRessurs());
+        settVisDokument(true);
+        axiosRequest<string, void>({
+            method: 'GET',
+            //TODO: remove mock
+            url: `/familie-ba-sak/api/journalpost/${journalPostId}/hent/${dokumentInfoId}`,
+            påvirkerSystemLaster: true,
+        })
+            .then((hentetDokumentData: Ressurs<string>) => {
+                if (hentetDokumentData.status === RessursStatus.SUKSESS) {
```

Comment:
If the fetch fails the iframe will display error in PDF. It is not a good error handling because it is moved from our new page where the error handling design has not been finalized.